DMD2_Timer.cpp

```diff
@@ -159,6 +166,50 @@ void BaseDMD::end()
   scanDisplay();
 }
 
+#elif defined(ESP32)
+hw_timer_t *timer = NULL;
+portMUX_TYPE timerMux = portMUX_INITIALIZER_UNLOCKED;
+
+void IRAM_ATTR esp32_ISR_wrapper(){
+  // claim interrupt handling
+  portENTER_CRITICAL_ISR(&timerMux);
+
+  scan_running_dmds();
+
+  // end interrupt handling
+  portEXIT_CRITICAL_ISR(&timerMux);
+}
+
+void BaseDMD::begin()
+{
+  beginNoTimer();
+
+  timer = timerBegin(ESP32_TIMER0, ESP32_TIMER0_PRESCALER, true);
```

Comment:
timer should be static to use two instances of DMD in the same time

```suggestion
  register_running_dmd(this);
  
  if (timer == NULL) {
    timer = timerBegin(ESP32_TIMER0, ESP32_TIMER0_PRESCALER, true);

    timerAttachInterrupt(timer, &esp32_ISR_wrapper, true);
    timerAlarmWrite(timer, ESP32_TIMER0_RELOAD_ON, true);

    // safety net
    yield();
    timerAlarmEnable(timer);
  }
```